modules/cpmso-yaml-failure-domain-gcp.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-configuration.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-failure-domain-gcp_{context}"]
+= Sample GCP failure domain configuration
+
+The control plane machine set concept of a failure domain is analogous to existing GCP concept of a link:https://cloud.google.com/compute/docs/regions-zones[_zone_]. The `ControlPlaneMachineSet` CR spreads control plane machines across multiple failure domains when possible.
+
+When configuring GCP failure domains in the control plane machine set, you must specify the zone name and the subnet to use.
+
+#This is the AWS content with "GCP" substituted for "AWS". I can't find what it should actually look like.#
```

Comment:
I dug around a bit but was not able to find what the failure domain will look like. Are you able to provide a sample?

Reply:
A sample of failure domain for GCP sent via Slack.